.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
```

Comment:
Since this is an older repo, we have a lot of backlogged dependabot updates. My thought was to include the dependabot yml from Web SDK that isolates the main/master branch,  limits updates to production only and runs once a week. This way we can reduce the deluge of PRrs and bring things in line until we can come up with a better solution.